packages/textkit/src/layout/wrapWords.js

```diff
@@ -49,7 +86,8 @@ const wrapWords = (engines = {}, options = {}, attributedString) => {
   for (let j = 0; j < words.length; j += 1) {
     const word = words[j];
     const parts = hyphenateWordWrapper(hyphenateWord, word)(word);
-    syllables.push(...parts);
+    const newParts = handleHyphenedWords(parts);
+    syllables.push(...newParts);
```

Comment:
shall we move this to line 88 and wrap the hyphenateWordWrapper function from handleHyphenedWords? 

then we don't need to assign this to a new variable